src/gtc/dace/dace_to_oir.py

```diff
@@ -74,7 +74,7 @@ def sdfg_arrays_to_oir_decls(sdfg: dace.SDFG) -> Tuple[List[oir.Decl], List[oir.
     return params, decls
 
 
-def convert(sdfg: dace.SDFG) -> oir.Stencil:
+def convert(sdfg: dace.SDFG, stencil: Optional[oir.Stencil] = None) -> oir.Stencil:
```

Comment:
This does not actually use stencil, only the attribute on stencil. Before merging, could you convert this to something like`*, loc: Optional[common.SourceLocation] = None`?

Reply:
sounds good 👍 will do